src/sentry/static/sentry/app/components/forms/selectControl.jsx

```diff
@@ -176,7 +176,7 @@ const SelectControl = props => {
   const mappedValue =
     props.multiple && Array.isArray(value)
       ? value.map(val => choicesOrOptions.find(option => option.value === val))
-      : choicesOrOptions.find(opt => opt.value === value);
+      : choicesOrOptions.find(opt => opt.value === value) || value;
```

Comment:
In alert rules, we need to be able to pass `null` because we never want the selector to actually change, we use it as more of a dropdown.